src/components/socio-carddesktop.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ColaboradorCardDesktop: React.FC = () => {
```

Comment:
To keep component names clean and coherent we should stick to english words. Also, since we probably will merge this component with the "Torna-te Sócio" one (since they're basically the same), we could just name it "JoinUsCard". That way, the name isn't specific to any purpose and the component can then receive a `type` property to inform what information it should display and change its text, colors, etc. accordingly.

Also, if I'm not mistaken, that `React.FC` type isn't needed.